man/baggr.Rd

```diff
@@ -36,7 +48,7 @@ If unspecified, the priors will be derived automatically based on data
 (and printed out in the console).}
 
 \item{prior_hypersd}{prior for hyper-standard deviation, used
-by Rubin and \code{"mutau"`` models; same rules apply as for }_hypermean`;}
+by Rubin and \verb{"mutau"`` models; same rules apply as for }_hypermean`;}
```

Comment:
Is this OK? I don't get what is going on here. (My old code also looks broken)

Reply:
This might have been a case where I accidentally deleted and re-added something and it got swept up in a commit unintentionally.

Reply:
Please check how it renders, I think my old code also looks wrong!